tests/osm/test_osm_utils.py

```diff
@@ -70,6 +69,7 @@ def test_filter_osm_defence(self):
         ):
             # type problems with bbox
             filter_osm(bbox=[0, 1.1, 0.1, 1.2])
+        # TODO: Add better testing for OSM on MacOS if possible
```

Comment:
Good to hear your thoughts on this one @CBROWN-ONS .

Reply:
Hi @r-leyshon . I can't quite remember too much about this, but I think when I was changing some of the tests, it appeared that some of the features within the `osm` module weren't being tested too thoroughly. Happy to jump on a call to discuss if you'd like.

Reply:
Comments captured in #191